sway/xdg_activation_v1.c

```diff
@@ -35,5 +35,6 @@ void xdg_activation_v1_handle_request_activate(struct wl_listener *listener,
 		return;
 	}
 
-	view_request_activate(view);
+	struct sway_seat *seat = (struct sway_seat *)event->token->seat->data;
```

Comment:
I think `event->token->seat` may be NULL here, if the client didn't set it.

Reply:
Ah, it seems like the next commit contains the logic to handle this case.

Reply:
I don't think the client has the option to create a token w/o a seat, as it is ultimately derived from the seat associated with the serial provided. The seat may still be null if it was destroyed after a token was granted. I was trying not to change behavior yet, but yeah the null check rightfully belongs in this commit. I'll move it here.

Reply:
The client does have the option to not specify the seat when creating the activation token, FWIW.

Reply:
For some reason I was under the impression wlroots would reject those but I think you're right. Should be good either way now anyhow.